flowmachine/flowmachine/utils.py

```diff
@@ -17,6 +17,31 @@
 from typing import Union, Tuple
 
 
+def get_stat(stat: str, column_name: str):
```

Comment:
Is it worth sticking a `valid_stats` set alongside this and importing it into all the queries that use `get_stat`, rather than duplicating the set of `valid_stats` everywhere?

Reply:
I was actually half wondering about making it an enum or something, but felt that might be overkill.

Reply:
I think it might be worth making it an enum. Particularly as several of the API-exposed query schemas use a `Statistic` field with a hard-coded list of valid stats, so it would be helpful to enforce consistency across the underlying flowmachine objects.